precipitation/ngsild-payloads/AirQualityObserved.json

```diff
@@ -0,0 +1,90 @@
+{
+  "id": "urn:ngsi-ld:AirQualityObserved:Pegomas",
+  "type": "AirQualityObserved",
+  "dateObserved": {
+    "type": "Property",
+    "value": "2016-03-15T11:00:00/2016-03-15T12:00:00"
+  },
+  "refCity": {
+    "type": "Relationship",
+    "object": "urn:ngsi-ld:City:Pegomas"
+},
+
+  "carbonMonoxide":[ 
+    {
+    "type": "Property",
+    "value": 500,
+    "unitCode": "GP",
+    "datasetId":"urn-ld:Dataset:carbonMonoxide:GP:APIX"
```

Comment:
where is the 'observedAt' for all properties. This is crucial to store temporal data. Without 'observedAt' new values will erase previous one without any storage.